### ISSUE TITLE
The integration dwd_weather is using the deprecated forecast attribute

### ISSUE THREAD

**Issue by heinemannj** [FL550/dwd_weather#75]
After upgrading of HA to

```
Home Assistant 2023.9.0
Supervisor 2023.08.3
Operating System 10.5
Frontend 20230906.1 - latest
```

the following waring pop's up:

`The integration dwd_weather is using the deprecated forecast attribute.`

Please check:

https://www.home-assistant.io/blog/2023/09/06/release-20239/#weather-forecast-service

**Comment by doktormerlin** [comment]
#74 already exists for this

**Comment by FL550** [comment]
Fixed in [v2.0.0](https://github.com/FL550/dwd_weather/releases/tag/v2.0.0).
